### ISSUE TITLE
TODO: Sign executables

### ISSUE THREAD

**Issue by seesharprun** [AzureCosmosDB/data-migration-desktop-tool#4]
The current GitHub actions release unsigned executables. The executables should be automatically signed using a code signing certificate.

**Comment by seesharprun** [comment]
See https://github.com/dotnet/sign

**Comment by philnach** [comment]
@markjbrown, @bowencode, I think this could be a good idea.  There's a free solution for Open Source projects: https://signpath.org. Would either of you object for me to make this happen.  Having code signing present will enable dmt to run on macOS and Windows without being blocked by the respective OS checkers.

**Comment by markjbrown** [comment]
Yes, that would be good as this needs to work smoothly for new users or users onboarding a new workload.  

@philnach I did a scan of our OSS docs on opensource.microsoft.com and cannot find anything that covers code signing so looks like no policies, etc. cover how or even if we do this. Would you mind taking a look at those docs too and be sure? I want to make sure we stay in compliance.
